angular/src/app/discovery/beacon/beacon-search-result-table/beacon-search-result-table.component.scss

```diff
@@ -1,7 +1,9 @@
 @import '~ddap-common-lib/styles/modules/colors';
+@import '~ag-grid-community/dist/styles/ag-theme-balham/vars/_ag-theme-balham-vars.scss';
 
 ddap-data-table {
   display: flex;
   flex-grow: 1;
   overflow-y: scroll;
+  border: 1px solid $ag-border-color;
```

Comment:
By the way, using this css variable from ag-grid gives a deprecation warning when building the front end. This is because the ag grid theme package uses deprecated sass syntax. I don't think it's a big deal for now, but we may need to find a new way of getting this color when we upgrade our sass version.